parsers/lib/quality.py

```diff
@@ -29,6 +29,10 @@ def validate_consumption(obj, zone_key):
     if obj['consumption'] is not None and obj['consumption'] < 0:
         raise ValidationError('%s: consumption has negative value '
                               '%s' % (zone_key, obj['consumption']))
+    # Plausibility Check, no more than 500GW
+    if obj['consumption'] is not None and abs(obj['consumption']) > 500000:
```

Comment:
You can do 
```suggestion
    if abs(obj.get('consumption', 0)) > 500000:
```
It basically will fallback to be 0